src/downloader/src/main/python/ModelDownloader.py

```diff
@@ -81,19 +87,57 @@ def _wrap(self, iter):
         return (ModelSchema.fromJava(s) for s in iter)
 
     def localModels(self):
+        """
+
+        Downloads models stored locally on the filesystem
+
+        """
         return self._wrap(self._model_downloader.localModels())
 
     def remoteModels(self):
+        """
+
+        Downloads models stored remotely.
+
+        """
         return self._wrap(self._model_downloader.remoteModels())
 
     def downloadModel(self, model):
+        """
```

Comment:
double-checking which newlines are needed for correct formatting and which can be eliminated without causing problems; will remove any that are unnecessary for doc format.